src/main/java/io/papermc/hangar/scanner/check/method/StringEncryptionCheck.java

```diff
@@ -12,6 +12,9 @@ public MethodCheckResult check(MethodInsnNode insnNode, MethodNode methodNode, C
         if (insnNode.name.equals("init") && insnNode.owner.equals("javax/crypto/Cipher")) {
             return new MethodCheckResult(Severity.MEDIUM, methodNode, classNode, "calls Cipher.init, an indicator for String Encryption");
         }
+        if (insnNode.name.equals("getInstance") && insnNode.owner.equals("javax/crypto/Cipher")) {
```

Comment:
not sure if we need two checks for basically the same thing, you cant use a cipher without calling init, can you?

Reply:
I can. I found it in the code